### DISCUSSION TITLE
And we're back!

### DISCUSSION THREAD

**Discussion by sahafeez** [kuestess/homebridge-platform-insteonlocal#256] [General]
https://arstechnica.com/gadgets/2022/06/insteon-smart-homes-resurrected-as-abruptly-as-they-were-bricked/

I am going to stick to my ARM system with the USB key and this software. Works better than the hub anyway.

**Comment by 64Spaces** [comment]
I agree. When the Insteon Cloud Services and Insteon App got turned back on, I used the opportunity to wipe and cleanup my Hub. And now that I’m running insteonlocal on a PLM and Raspberry Pi, it’s performing perfectly. Hopefully, we will see new hardware solutions as the chip shortages get better over the next year.

**Comment by kuestess** [comment]
Hopefully this new owner embraces developers - their documentation (or lack of) makes things very difficult.
